packages/admin-ui/ui/src/hooks/use-edit-product-actions.tsx

```diff
@@ -132,6 +132,7 @@ const useEditProductActions = (productId: string) => {
     const newStatus = currentStatus === "published" ? "draft" : "published"
     updateProduct.mutate(
       {
+        // @ts-ignore TODO fix update type in API
```

Comment:
what do you mean?

Reply:
I am reverting the changes, but I believe this is due to this Typescript error related to assigning a string to an enum export from `@medusajs/medusa`: 
![image](https://github.com/medusajs/medusa/assets/59018053/0c155810-2554-4c11-a053-6fc776977fc8)